Frontend/src/app/views/dashboard/project-details/project-details.component.ts

```diff
@@ -173,35 +173,35 @@ export class ProjectDetailsComponent {
   sendArchiveRequest() {
     this.archiving = true;
 
-    let requestDetails : ArchiveProjectDTO = {
+    let requestDetails: ArchiveProjectDTO = {
```

Comment:
revert formatting

Reply:
Sorry about the formatting i think it's something to do with my push to git thats formatting because I didnt format anything before either and the details page had alot of format changes.